analysis/diabetes_analysis.ipynb

```diff
@@ -223,8 +226,337 @@
    ],
    "source": [
     "# load data\n",
-    "df = pd.read_csv('../data/diabetes.csv')\n",
-    "df.head()"
+    "df_original = pd.read_csv('../data/diabetes.csv')\n",
+    "df_original.head()"
```

Comment:
would recommend use code `df_original` directly (WITHOUT the `.head()`) so that the output shows the default 10 rows but more importantly shows teh number of rows x columns. In that way, it will be an easy comparison to the data after `pandera`

Reply:
Yes that makes sense I will remove the head() method